src/othermbc.md

```diff
@@ -40,70 +41,75 @@ nonsensical.
 
 The mapper does not support an outer bank for battery SRAM.
 
-To start a game, do the following in code run from RAM: Write $A5 to
-$1000, write game starting bank number to $2000, write any value to
-$7000, write $98 to $1000, write $01 to $2000 (so that 32K games
-work), jump to $0100.
+To start a game, do the following with code run from RAM:
+1. Write $A5 to $1000
+2. Write game's first bank number to $2000
+3. Write any value to $7000
+4. Write $98 to $1000
+5. Write $01 to $2000 (so that 32K games work)
+6. Jump to $0100
 
 ## Wisdom Tree
 
-The Wisdom Tree mapper is a simple, cost-optimized one chip design
-consisting of a 74LS377 octal latch, aside from the ROM chip. Because
+The Wisdom Tree mapper is a simple, cost-optimized one-chip design
+consisting of a 74LS377 octal latch in addition to the ROM chip. Because
 the mapper consists of a single standard 74 series logic chip, it has
 two unusual properties:
 
-First, unlike a usual MBC, it switches the whole 32 kiB ROM area instead
-of just the $4000-$7FFF area. If you want to use the interrupt vectors
+First, unlike a usual MBC, it switches the whole 32 KiB ROM area instead
+of just the $4000-$7FFF area. Therefore, if you want to use [the interrupt vectors](<#Interrupt Handling>)
 with this cart, you should duplicate them across all banks.
 Additionally, since the initial state of the '377 can't be guaranteed,
```

Comment:
What is the "initial state of the '377'? We are referring to the  74LS377 but it's not clear what the "initial state" consists of